databases/week1/databases.sql

```diff
@@ -0,0 +1,233 @@
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (30, 'Remove facebook from phone', 'To avoid interruptions when working', '2017-10-26 17:15:07', '2017-10-13 03:36:47', '2017-12-19 11:10:02', 3, 4);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (31, 'Backup databases to external disk', 'Remember to store the disk in another physical location', '2017-09-09 17:32:33', '2017-10-01 21:18:59', '2017-12-23 14:21:01', 1, 2);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (32, 'Put up the new lamp in the hallway', NULL, '2017-10-15 05:45:54', '2017-10-16 14:05:35', '2017-12-29 02:29:26', 3, 3);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (33, 'Hang up paintings in living room', NULL, '2017-09-10 05:36:11', '2017-10-09 17:40:42', null, 3, 4);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (34, 'Buy plane ticket to Auckland', 'Check prices online first!', '2017-09-05 09:07:22', '2017-10-15 09:36:06', '2017-12-07 11:10:05', 1, 9);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (35, 'Learn about NoSQL databases', 'MongoDB, CouchDB, etc.', '2017-10-20 01:41:53', '2017-10-04 07:19:56', '2017-12-23 10:13:42', 2, NULL);
+
+
+-- HOMEWORK
+
+-- Find out how many tasks are in the task table:
+
+SELECT * id FROM task;
+
+-- Find out how many tasks in the task table do not have a valid due date:
+
+SELECT * FROM task WHERE due_date='NULL';
+
+-- Find all the tasks that are marked as done
+
+SELECT * FROM task WHERE status_id='3';
+
+-- Find all the tasks that are not marked as done
+
+SELECT * FROM task WHERE status_id='1' OR status_id='2';
+
+-- Get all the tasks, sorted with the most recently created first
+
+SELECT * FROM task ORDER BY  created ASC;
+
+-- Get the single most recently created task
+
+SELECT * FROM task WHERE created ASC;
+
+-- Get the title and due date of all tasks where the title or description contains database
+
+SELECT title , due_date  FROM task WHERE title OR description LIKE '%database%'
+
+-- Get the title and status (as text) of all tasks
+
+SELECT title AS title_Activity ,
+
+ status AS status_Funtion FROM task 
+
+JOIN status ON status.id = task.status_id;
+
+-- Get the name of each status, along with a count of how many tasks have that status
+
+SELECT status AS status_task .
+
+ COUNT(task.tittle)
+FROM task
+
+Join status ON staus.id = task.status_id GROUP BY status_id;
+
+
+-- Get the names of all statuses, sorted by the status with most tasks first
+
+SELECT * FROM name ORDER BY statues, INNER JOIN 
```

Comment:
There is a mistake here, when you write, SELECT * FROM name after FROM you should write the name of table you are selecting something.

Reply:
Alex Ekamba i have added the changes now, thanks for help..